components/Collections/src/private/MDCCollectionInfoBarView.m

```diff
@@ -175,13 +182,22 @@ - (void)setStyle:(MDCCollectionInfoBarViewStyle)style {
 
     // Adds border to be positioned during sublayer layout.
     self.backgroundView.clipsToBounds = YES;
-    if (!_backgroundBorderLayer) {
-      _backgroundBorderLayer = [CALayer layer];
-      _backgroundBorderLayer.borderColor = [UIColor colorWithWhite:0 alpha:(CGFloat)0.1].CGColor;
-      _backgroundBorderLayer.borderWidth = 1 / [[UIScreen mainScreen] scale];
-      [self.backgroundView.layer addSublayer:_backgroundBorderLayer];
+  }
+  [self updateBorderColor];
+}
+
+- (void)updateBorderColor {
+  CGFloat white = 0.0f;
```

Comment:
Please use either an explicit cast to `(CGFloat)` or use integral values (my preferred option).

```objc
CGFloat white = 0;
white = 1;

// Alternatively
CGFloat white = (CGFloat)0.0;
white = (CGFloat)1.0;
```